### DISCUSSION TITLE
Get some weird errors in authelia logs

### DISCUSSION THREAD

**Discussion by cyb3rgh05t** [authelia/authelia#6658] [Q&A] [answered]
Hi,

since some days i get some weird errors in authelia logs. (see atteched log file)
What could cause this error?
[authelia-2024-02-06T18-51-01.log.gz](https://github.com/authelia/authelia/files/14184917/authelia-2024-02-06T18-51-01.log.gz)

thanks in advance

**Comment by james-d-elliott** [comment, marked as answer]
Hey sorry this is an issue with the session lib we use. It's been somewhat fixed and soon to be released see [here](https://github.com/authelia/authelia/issues/3751) but we're looking to move away from it as well.

**Reply by james-d-elliott** [reply]
~~Hold on, thought I found the issue, but I found another.~~ Updated.

**Comment by cyb3rgh05t** [comment]
thanks
